share/nisar/defaults/gcov.yaml

```
@@ -191,8 +191,10 @@ runconfig:
 
                 # Flag indicating whether the output GCOV product should include
                 # off-diagonal covariance elements. If False, only the diagonal
-                # elements are included.
-                fullcovariance:   False
+                # elements are included. If empty, `fullcovariance` is set to
+                # `True` any frequency to be processed includes a full-pol dataset
+                # (3 or 4 polarizations), and `False` otherwise
```

Suggest copying some of the description from the comments added in `gcov_runconfig.py` here to help explain the behavior to users.

One minor addition would be to try to include some of the verbiage from the XML file to better explain the receive-only case to users who might not be familiar with that mode.

https://github-fn.jpl.nasa.gov/NISAR-ADT/NISAR_PIX/blob/9f8c22fd4b18016b12d15c52ecf3956f5fc649ae/XML/L1/nisar_L1_RSLC.xml#L333

```suggestion
                # elements are included. If empty, `fullcovariance` is set to
                # `True` any frequency to be processed includes a full-pol dataset
                # (i.e. includes both co-pols and at least one cross-pol), and
                # `False` otherwise. 
```